### DISCUSSION TITLE
Possible to connect to redis KV via gui?

### DISCUSSION THREAD

**Discussion by mcmadafly** [vercel/community#2338] [Help] [answered]
### Summary

Is it possible to connect to a KV instance with a Redis GUI?

I've tried connecting to Redis / KV with both TablePlus and Medis but it returns `Server closed the connection`.

<img width="520" alt="Screenshot 2023-05-06 at 2 01 49 PM" src="https://user-images.githubusercontent.com/454117/236642115-0edfb71f-de62-44d1-875b-da58cb422042.png">

**Comment by TooTallNate** [comment]
With `redis-cli`, you need to use the `--tls` flag. So try enabling that "SSL mode" for your GUI app, as that should be related.

**Reply by mcmadafly** [reply]
@TooTallNate - Didn't work. 😢  Maybe I'm missing something? Here's what I tried:

<img width="527" alt="Screenshot 2023-05-07 at 6 39 43 PM" src="https://user-images.githubusercontent.com/454117/236708270-71fe43af-0667-4219-9159-89962f02f627.png">

**Reply by franknoel** [reply, marked as answer]
It worked for me in Redis Insight, thank you.

**Reply by mcmadafly** [reply]
> It worked for me in Redis Insight, thank you.

Thanks I'll give it a try!

**Reply by luin** [reply]
Just a reference for anyone who encounter this: if you are using Medis, you can enable SSL mode by checking the "Enable SSL" checkbox:

![CleanShot 2024-01-09 at 09 09 05@2x](https://github.com/vercel/community/assets/635902/ce3878d4-b57d-4b4e-82b7-718225cba97b)
